plugins/gradle/src/main/java/org/wildfly/swarm/plugin/gradle/PackageTask.java

```diff
@@ -66,6 +71,23 @@ public void packageForSwarm() throws Exception {
 
         GradleArtifactResolvingHelper resolvingHelper = new GradleArtifactResolvingHelper(project);
         Properties propertiesFromExtension = getPropertiesFromExtension();
+        List<File> moduleDirs = getModuleDirs();
+        if (moduleDirs.isEmpty()) {
```

Comment:
Looking at the deleted code, it seems the original indention was to _add_ the `modules` dir, if it exists, to the configured ones. But here you use it only if there's no configured module directory. Is that ok?